.gitignore

```
@@ -13,3 +13,6 @@ pyvenv.cfg
 share/
 .idea/
 .tox/
+
+requirements.txt
+requirements-last-build.txt
```

```suggestion
/requirements.txt
/requirements-last-build.txt
```
nit: if you have other charms (e.g. test charms) in this repo, I imagine you'd use requirements.txt instead of poetry for those charms (no need for dep locking imo) so you'd probably want to version control those requirements.txt files